translate/gast_to_js.py

```diff
@@ -0,0 +1,65 @@
+
+
+def list_to_csv_str(l):
+    s = ""
+    for i in l:
+        s += str(i) + ", "
+    
+    return s[:-2] # remove last comma and space
```

Comment:
Might be easier to use [join](https://www.tutorialspoint.com/python/string_join.htm) to avoid  this issue entirely.